Mypy check don't pass after release 0.1.5.

**Describe the bug**

Mypy checks don't pass after release `0.1.5`.

**To Reproduce**
Steps to reproduce the behaviour:
1. `git clone https://github.com/fetchai/agents-aea/`
2. `cd agents-aea`
3. `git checkout v0.1.5`
4. `tox -e mypy`

**Expected behavior**

Mypy checks should pass.

**Actual behaviour**

```

(agents-aea) marcofavorito@marco-GL502VMK:~/workfolder/agents-aea$ tox -e mypy
mypy create: /home/marcofavorito/workfolder/agents-aea/.tox/mypy
mypy installdeps: mypy
WARNING: test command found but not installed in testenv
  cmd: /home/marcofavorito/.virtualenvs/agents-aea-jzIedExH/bin/python
  env: /home/marcofavorito/workfolder/agents-aea/.tox/mypy
Maybe you forgot to specify a dependency? See also the whitelist_externals envconfig setting.

DEPRECATION WARNING: this will be an error in tox 4 and above!
mypy installed: actionid: mypy,msg: envreport,cmdargs: '/home/marcofavorito/.virtualenvs/agents-aea-jzIedExH/bin/python3.7 -m pipenv graph',,base58==1.0.3,click-log==0.3.2,  - click [required: Any, installed: 7.0],colorlog==4.0.2,cryptography==2.7,  - asn1crypto [required: >=0.21.0, installed: 0.24.0],  - cffi [required: >=1.8,!=1.11.3, installed: 1.12.3],    - pycparser [required: Any, installed: 2.19],  - six [required: >=1.4.1, installed: 1.12.0],docker==4.0.2,  - requests [required: >=2.14.2,!=2.18.0, installed: 2.22.0],    - certifi [required: >=2017.4.17, installed: 2019.9.11],    - chardet [required: >=3.0.2,<3.1.0, installed: 3.0.4],    - idna [required: >=2.5,<2.9, installed: 2.8],    - urllib3 [required: >=1.21.1,<1.26,!=1.25.1,!=1.25.0, installed: 1.25.6],  - six [required: >=1.4.0, installed: 1.12.0],  - websocket-client [required: >=0.32.0, installed: 0.56.0],    - six [required: Any, installed: 1.12.0],flake8-docstrings==1.4.0,  - flake8 [required: >=3, installed: 3.7.8],    - entrypoints [required: >=0.3.0,<0.4.0, installed: 0.3],    - mccabe [required: >=0.6.0,<0.7.0, installed: 0.6.1],    - pycodestyle [required: >=2.5.0,<2.6.0, installed: 2.5.0],    - pyflakes [required: >=2.1.0,<2.2.0, installed: 2.1.1],  - pydocstyle [required: >=2.1, installed: 3.0.0],    - six [required: Any, installed: 1.12.0],    - snowballstemmer [required: Any, installed: 1.9.1],jsonschema==3.0.2,  - attrs [required: >=17.4.0, installed: 19.1.0],  - pyrsistent [required: >=0.14.0, installed: 0.15.4],    - six [required: Any, installed: 1.12.0],  - setuptools [required: Any, installed: 41.2.0],  - six [required: >=1.11.0, installed: 1.12.0],mkdocs-material==4.4.2,  - mkdocs [required: >=1, installed: 1.0.4],    - click [required: >=3.3, installed: 7.0],    - Jinja2 [required: >=2.7.1, installed: 2.10.1],      - MarkupSafe [required: >=0.23, installed: 1.1.1],    - livereload [required: >=2.5.1, installed: 2.6.1],      - six [required: Any, installed: 1.12.0],      - tornado [required: Any, installed: 6.0.3],    - Markdown [required: >=2.3.1, installed: 3.1.1],      - setuptools [required: >=36, installed: 41.2.0],    - PyYAML [required: >=3.10, installed: 5.1.2],    - tornado [required: >=5.0, installed: 6.0.3],  - mkdocs-minify-plugin [required: >=0.2, installed: 0.2.1],    - htmlmin [required: >=0.1.4, installed: 0.1.12],    - jsmin [required: >=2.2.2, installed: 2.2.2],    - mkdocs [required: >=1.0.4, installed: 1.0.4],      - click [required: >=3.3, installed: 7.0],      - Jinja2 [required: >=2.7.1, installed: 2.10.1],        - MarkupSafe [required: >=0.23, installed: 1.1.1],      - livereload [required: >=2.5.1, installed: 2.6.1],        - six [required: Any, installed: 1.12.0],        - tornado [required: Any, installed: 6.0.3],      - Markdown [required: >=2.3.1, installed: 3.1.1],        - setuptools [required: >=36, installed: 41.2.0],      - PyYAML [required: >=3.10, installed: 5.1.2],      - tornado [required: >=5.0, installed: 6.0.3],  - Pygments [required: >=2.2, installed: 2.4.2],  - pymdown-extensions [required: >=4.11, installed: 6.1],    - Markdown [required: >=3.0.1, installed: 3.1.1],      - setuptools [required: >=36, installed: 41.2.0],    - pep562 [required: Any, installed: 1.0],mypy==0.730,  - mypy-extensions [required: >=0.4.0,<0.5.0, installed: 0.4.1],  - typed-ast [required: >=1.4.0,<1.5.0, installed: 1.4.0],  - typing-extensions [required: >=3.7.4, installed: 3.7.4],oef==0.6.10,  - graphviz [required: Any, installed: 0.13],  - protobuf [required: Any, installed: 3.9.2],    - setuptools [required: Any, installed: 41.2.0],    - six [required: >=1.9, installed: 1.12.0],pytest-cov==2.7.1,  - coverage [required: >=4.4, installed: 4.5.4],  - pytest [required: >=3.6, installed: 5.1.3],    - atomicwrites [required: >=1.0, installed: 1.3.0],    - attrs [required: >=17.4.0, installed: 19.1.0],    - importlib-metadata [required: >=0.12, installed: 0.23],      - zipp [required: >=0.5, installed: 0.6.0],        - more-itertools [required: Any, installed: 7.2.0],    - more-itertools [required: >=4.0.0, installed: 7.2.0],    - packaging [required: Any, installed: 19.2],      - pyparsing [required: >=2.0.2, installed: 2.4.2],      - six [required: Any, installed: 1.12.0],    - pluggy [required: >=0.12,<1.0, installed: 0.13.0],      - importlib-metadata [required: >=0.12, installed: 0.23],        - zipp [required: >=0.5, installed: 0.6.0],          - more-itertools [required: Any, installed: 7.2.0],    - py [required: >=1.5.0, installed: 1.8.0],    - wcwidth [required: Any, installed: 0.1.7],tox-pipenv==1.9.0,  - pipenv [required: >=2018.05.18, installed: 2018.11.26],    - certifi [required: Any, installed: 2019.9.11],    - pip [required: >=9.0.1, installed: 19.2.3],    - setuptools [required: >=36.2.1, installed: 41.2.0],    - virtualenv [required: Any, installed: 16.7.5],    - virtualenv-clone [required: >=0.2.5, installed: 0.5.3],  - tox [required: >=3.0.0, installed: 3.7.0],    - filelock [required: >=3.0.0,<4, installed: 3.0.12],    - pluggy [required: >=0.3.0,<1, installed: 0.13.0],      - importlib-metadata [required: >=0.12, installed: 0.23],        - zipp [required: >=0.5, installed: 0.6.0],          - more-itertools [required: Any, installed: 7.2.0],    - py [required: >=1.4.17,<2, installed: 1.8.0],    - setuptools [required: >=30.0.0, installed: 41.2.0],    - six [required: >=1.0.0,<2, installed: 1.12.0],    - toml [required: >=0.9.4, installed: 0.10.0],    - virtualenv [required: >=1.11.2, installed: 16.7.5],
mypy run-test-pre: PYTHONHASHSEED='1820354607'
mypy runtests: PYTHONHASHSEED=None
mypy runtests: commands[0] | mypy aea packages tests scripts
aea/__init__.py:27: error: Argument 1 to "getfile" has incompatible type "Optional[FrameType]"; expected "Union[Module, Type[Any], MethodType, FunctionType, TracebackType, FrameType, CodeType]"
aea/configurations/loader.py:32: error: Argument 1 to "getfile" has incompatible type "Optional[FrameType]"; expected "Union[Module, Type[Any], MethodType, FunctionType, TracebackType, FrameType, CodeType]"
tests/conftest.py:36: error: Argument 1 to "getfile" has incompatible type "Optional[FrameType]"; expected "Union[Module, Type[Any], MethodType, FunctionType, TracebackType, FrameType, CodeType]"
Found 3 errors in 3 files (checked 119 source files)
ERROR: InvocationError for command '/home/marcofavorito/.virtualenvs/agents-aea-jzIedExH/bin/python3.7 -m pipenv run mypy aea packages tests scripts' (exited with code 1)
_________________________________________________________________________________________________ summary __________________________________________________________________________________________________
ERROR:   mypy: commands failed
```

If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04.3 LTS
 - Browser: N/A
 - Version: `0.1.5`

**Additional context**

I'd suggest fixing the dependencies versions, both in `Pipfile` and in `tox.ini` (in the latter the dependencies could be installed with `pipenv`, for every test environments.)

--- a/aea/__init__.py
+++ b/aea/__init__.py
@@ -24,4 +24,4 @@ import os
 from aea.__version__ import __title__, __description__, __url__, __version__
 from aea.__version__ import __author__, __license__, __copyright__
 
-AEA_DIR = os.path.dirname(inspect.getfile(inspect.currentframe()))
+AEA_DIR = os.path.dirname(inspect.getfile(inspect.currentframe()))  # type: ignore

--- a/aea/configurations/loader.py
+++ b/aea/configurations/loader.py
@@ -29,7 +29,7 @@ from jsonschema import validate
 
 from aea.configurations.base import AgentConfig, SkillConfig, ConnectionConfig
 
-_CUR_DIR = os.path.dirname(inspect.getfile(inspect.currentframe()))
+_CUR_DIR = os.path.dirname(inspect.getfile(inspect.currentframe()))  # type: ignore
 _SCHEMAS_DIR = os.path.join(_CUR_DIR, "schemas")
 
 T = TypeVar('T', AgentConfig, SkillConfig, ConnectionConfig)

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -33,7 +33,7 @@ from oef.agents import AsyncioCore, OEFAgent
 
 logger = logging.getLogger(__name__)
 
-CUR_PATH = os.path.dirname(inspect.getfile(inspect.currentframe()))
+CUR_PATH = os.path.dirname(inspect.getfile(inspect.currentframe()))  # type: ignore
 ROOT_DIR = os.path.join(CUR_PATH, "..")
 
 CONFIGURATION_SCHEMA_DIR = os.path.join(ROOT_DIR, "aea", "configurations", "schemas")

